app/login.js

```diff
@@ -83,42 +40,59 @@ define(['jquery', 'app/config', 'app/database', 'parsley', 'app/forms','app/user
 	}
 
 	function handleLogin() {
-		var result = getUser();
-		user.setCookie(result);
-		window.location = "index.html?message=login";
+		var username = $("#username").val().toLowerCase();
+		var password = $("#password").val();
+		var field;
+		if (db.online()) {
+			db.getOnlineQuery("getUserbyUsername", [username], function(result) {
```

Comment:
The code after if (db.online()) is now been put into seperate functions. This is what you meant with this comment right?